debugging-with-ssh-access.md

```diff
@@ -0,0 +1,81 @@
+Often the best way to troubleshoot failures in your pipelines is to SSH into a
+job and inspect log files, running processes, and directory paths. Semaphore 2.0
+gives you the option to access all running jobs via SSH, to restart your jobs
+in debug mode, or to start on-demand virtual machines to explore the build
+environment.
+
+* [Explore the build environment](#explore-the-build-environment)
+* [Inspect the state of a running job](#inspect-the-state-of-a-running-job)
+* [Restart a job in debug mode](#restart-a-job-in-debug-mode)
+* [Port forward your web server and debug UI issues](#port-forward-your-web-server-and-debug-ui-issues)
+
+## Explore the build environment
+
+Setting up a pipeline can be challenging if you are not familiar with the
+software stack installed in Semaphore's virtual machines. Starting a debug
+session for your project is a great place to start exploring.
+
+To start a debug session for your project, run:
+
+``` bash
+sem debug project <name-of-your-project>
+```
+
+The above command will start a virtual machine connected with your git
+repository and attach you to it via an SSH session.
+
+## Inspect the state of a running job
```

Comment:
I would just mention here that they can get a list of running jobs by running `sem get jobs`. I think it would be great usability addition to this section. In that case they really don't need to read any other page to get 98% of all debugging goodness.

Reply:
I wanted to avoid making these into a long paragraph that explain details. But I agree with this. 

I'll mention `sem get jobs` as a "quick way to find your job" sentence at the bottom.